From 44a4c50f2cdd3ddde06289ab51636c16c9ba5c0f Mon Sep 17 00:00:00 2001
From: Michiel Beijen <michiel.beijen@gmail.com>
Date: Thu, 18 Aug 2016 09:55:25 +0200
Subject: [PATCH] Compiling w/libmariadb disabled SSL capacity

By mistake, when DBD::mysql was compiled against libmariadb, it
did not compile in SSL support.

Reported by Jitka Plesnikova @ Red Hat
  (https://rt.cpan.org/Public/Bug/Display.html?id=116959)
  (https://bugzilla.redhat.com/show_bug.cgi?id=1366773)
---
 Makefile.PL             | 4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 1284736..6b7d12e 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -680,8 +680,8 @@ section "Linker flags" or type
 MSG
     }
     elsif ($param eq "nossl") {
-      $opt->{$param} = ($opt->{"libs"} =~ /ssl/) ? 1 : 0;
-      $source->{$param} = $opt->{$param} ? "choice" : "default";
+      $source->{$param} = $opt->{$param} ? "User's choice" : "default";
+      $opt->{$param} = $opt->{$param} || 0;
     }
     elsif ($param eq "ssl") {
         # default, for legacy reasons
